src/main/java/org/bytedeco/javacpp/tools/Generator.java

```diff
@@ -4386,7 +4385,7 @@ String[] cppTypeName(Class<?> type, Annotation[] annotations) {
         } else if (type.isPrimitive()) {
             prefix = type.getName();
         } else if (FunctionPointer.class.isAssignableFrom(type)) {
-            Method[] functionMethods = functionMethods(type, null);
+            Method[] functionMethods = functionMethods(type, type.getDeclaredMethods(), null);
```

Comment:
Doesn't that just move the call to getDeclaredMethods() outside of functionMethods()? Why does this fix anything?

Reply:
Ah, it's not for this call, but the one above, got it.